PopApp/Components/CreateMeeting.js

```diff
@@ -75,6 +95,9 @@ const CreateMeeting = () => {
     if (event.type === 'set' && mode === 'date') {
       showTimepicker();
     }
+    if (event.type === 'set' && mode === 'time' && finishDate && finishDate <= startDate) {
```

Comment:
Just want to double check, finishDate should be less than or equal to startDate?

Reply:
Less than because I don't want the user to set a finishDate smaller than the startDate